distelli-manifest.yml

```diff
@@ -0,0 +1,25 @@
+team-modules/puppetlabs-motd:
```

Comment:
project name needs changed

Reply:
@sheenaajay just mentioned that to me :grimacing: